scripts/g_r.py

```diff
@@ -0,0 +1,60 @@
+import datetime
+import pandas
+from collections import Counter
+from sqlalchemy import create_engine
+engine = create_engine('mysql://root@localhost:3306/pantherbot_test', echo=False)
+
+def g_r(response, args):
+    if args[0] == 'time':
+        if len(args)!=4:
+            return ["ERROR: Please use the following syntax `time <channel> 2/1/17 5/1/18`"]
+        return time(args[2::], args[1])
+
+    if args[0] == 'top_users':
+        pass
+
+    if args[0] == 'emoji':
+        pass
+
+    if args[0] == 'channel':
```

Comment:
Can we edit the PR to reflect this then?

> Collects data from events regarding messages, emojis, team joins, channel creations, archivals
Is kinda miss leading as it implies these features are done, unless the data is collected then can we atleast not that their relevant functions aren't ready?